libs/hdf_converters/src/nessus-mapper.ts

```diff
@@ -143,14 +143,18 @@ function getStig(item: unknown): string {
   }
 }
 function getStatus(item: unknown): ExecJSON.ControlResultStatus {
-  if (_.has(item, 'cm:compliance-result')) {
-    if (_.get(item, 'cm:compliance-result') === 'PASSED') {
+  const result = _.get(item, 'cm:compliance-result');
+  switch (result) {
+    case 'PASSED':
       return ExecJSON.ControlResultStatus.Passed;
-    } else {
+    case 'WARNING':
+      return ExecJSON.ControlResultStatus.Skipped;
+    case 'ERROR':
+      return ExecJSON.ControlResultStatus.Error;
+    case 'FAILED':
+      return ExecJSON.ControlResultStatus.Failed;
+    default:
       return ExecJSON.ControlResultStatus.Failed;
```

Comment:
If Failed is the default then there is no need to include the Failed case as well here.

Reply:
Ok, I'll make the change